docs_src/getting-started/Ch-GettingStartedUsersNexus.md

```diff
@@ -1,95 +1,45 @@
 
-Docker Compose files that pull the latest EdgeX container images from
-Nexus are available here:
-<https://github.com/edgexfoundry/developer-scripts/tree/master/releases/nightly-build/compose-files>.
+- your hardware (x86 or ARM)
+- the database you want to use (Mongo or Redis)
+- your desire to have security services on or off
 
-If you are creating your own Docker Compose file or want to use an
-existing EdgeX Docker Compose file but selectively use Nexus images,
-replace the name/location of the Docker image in your docker-compose.yml
-file for the containers you want to get from Nexus versus Docker Hub.
-For example, the config-seed item in docker-compose.yml might ordinarily
-look like this:
+![image](EdgeX_GettingStartedNexusCompose.png)
 
-    config-seed:
-      **image: edgexfoundry/docker-core-config-seed**
+!!! Warning
+    The "Nightly Build" images are provided as is and may not always function properly or with other EdgeX services.  Use with caution and typically only if you are a developer/contributor to EdgeX. These images represent the latest development work and may not have been thoroughly tested or integrated.
 
-      ports:
-          \- "8400:8400"\
-          \- "8500:8500"\
-          \- "8600:8600"\
+## Using Nexus Images
+The operations to pull the images and run the containers from the containers is the same as when using EdgeX images from Docker Hub (see [Getting Started with Docker](./Ch-GettingStartedUsers.md#run-edgex-foundry)).
 
-      container_name: edgex-config-seed
+To get containers from the Nexus Repository, in a command terminal, change directories to the location of your downloaded Nexus Docker Compose yaml.  Rename the file to docker-compose.yml.  Then run the following command in the terminal to pull (fetch) and then start the EdgeX Nexus-image containers.
 
-      hostname: edgex-core-config-seed
+``` bash
+docker-compose up -d
+```
+steps.
 
-      networks:
-        edgex-network:
-          aliases:
-              \- edgex-core-consul\
-      volumes_from:
-        \- volume\
-      depends_on:
-        \- volume\
+## Using a Single Nexus Image
+In some cases, you may only need to use a single image from Nexus while other EdgeX services are created from the Docker Hub images.  In this case, you can simply replace the image location for the selected image in your original Docker Compose file.  The address of Nexus is nexus3.edgexfoundry.org at port 10004.  So, if you wished to use the EdgeX core data image from Nexus, you would replace the name and location of the core data image "edgexfoundry/docker-core-data-go:1.2.1" with "nexus3.edgexfoundry.org:10004/docker-core-data-go:master" in the Compose file.
```

Comment:
To me - as a developer - This is the single most important line in this entire document and its buried way down here:
"The address of Nexus is nexus3.edgexfoundry.org at port 10004." 
It needs a code format block and needs to be "Above the fold" -- ideally first section. Something like:


# Getting Docker Images from EdgeX Nexus Repository

Released EdgeX Docker container images are available from [Docker Hub](https://hub.docker.com/search?q=edgexfoundry&type=image).  Pre-release/WIP Docker Images are built nightly and are hosted in the following repository: 
``` bash
nexus3.edgexfoundry.org:10004
```

In some cases, it may be necessary to get your EdgeX container images from the Nexus repository which is managed by the Linux Foundation.

Reply:
agree - add this to the top of the file